common/errors.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/hashicorp/errwrap"
+)
+
+// ErrorSanitizer is designed to handle
+type ErrorSanitizer struct {
+	output io.Writer
+	prefix string
+	debug  bool
+}
+
+func NewErrorSanitizer(out io.Writer, prefix string, debug bool) *ErrorSanitizer {
+	return &ErrorSanitizer{
+		output: out,
+		prefix: prefix,
+		debug:  debug,
+	}
+}
+
+// Print prints the msg and its error messages to the designated io.Writer.
+func (es *ErrorSanitizer) Print(msg string, e error) {
+	output := es.output
+	if output == nil {
```

Comment:
If it's not properly initialized. But I guess since we've got full control here we can get rid of this basic check.